app/questionnaire/navigator.py

```diff
@@ -101,6 +101,11 @@ def __init__(self, survey_json, metadata=None, answer_store=None):
         self.metadata = metadata or {}
         self.survey_json = survey_json
 
+        self.preceeding_path = []
```

Comment:
Think I'll defer to @ajmaddaford on this as he had some thoughts on why interstitials should be handled differently.